tests/test_assets.py

```diff
@@ -308,28 +307,20 @@ def test_lookup_symbol_fuzzy(self):
             finder.retrieve_asset(i) for i in range(3)
         )
 
-        for i in range(2):  # we do it twice to test for caching bugs
+        # we do it twice to catch caching bugs
+        for i in range(2):
+            # Shouldn't find this with no fuzzy_str passed.
             self.assertIsNone(finder.lookup_symbol('test', as_of))
-            self.assertEqual(
-                asset_1,
-                finder.lookup_symbol('test@1', as_of)
-            )
+            self.assertIsNone(finder.lookup_symbol('test1', as_of))
+            self.assertEqual(asset_1, finder.lookup_symbol('test.1', as_of))
 
             # Adding an unnecessary fuzzy shouldn't matter.
-            self.assertEqual(
-                asset_1,
-                finder.lookup_symbol('test@1', as_of, fuzzy=True)
-            )
+            self.assertEqual(asset_1, finder.lookup_symbol('test/1', as_of))
 
-            # Shouldn't find this with no fuzzy_str passed.
-            self.assertIsNone(finder.lookup_symbol('test1', as_of))
             # Should find exact match.
-            self.assertEqual(
-                asset_1,
-                finder.lookup_symbol('test1', as_of, fuzzy=True),
-            )
+            self.assertEqual(asset_1, finder.lookup_symbol('test-1', as_of))
```

Comment:
I don't know that we need multiple tests, or if we do, maybe we should run each of the possible fuzzy characters in a nice comprehension.

Additionally, we should have a test for a fuzzy symbol that doesn't exist within the list and test that nothing is returned. As well as having one with two delimiters, to check that it's doing only one mapping.